couchbase/search/v1/search.proto

```diff
@@ -208,6 +208,43 @@ message Sorting {
   }
 }
 
+message DateRange {
+  string name = 1;
+  optional string start = 2;
+  optional string end = 3;
+}
+
+message DateRangeFacet {
+  string field = 1;
+  uint32 size = 2;
+  repeated DateRange date_ranges = 3;
+}
+
+message NumericRange {
+  string name = 1;
+  optional float min = 2;
```

Comment:
Not really sure how specify "one of min, max, or both"

Reply:
I think the answer is that both are optional, but we return IllegalArgument if neither was specified.